.sail.yml

```diff
@@ -0,0 +1,70 @@
+workflow:
+  - install
+  - sail:parallel:
+    - flow
+    - lint
+    - build
+    - testWeb
+    - testNative
+    - testPrimitives
+    - testIntegration
+  - size
+tasks:
+  install:
+    image: philpl/alpine-chrome-node:latest
+    command:
+      - yarn
+    env:
+      - GOOGLE_CHROME_BINARY: "/usr/bin/chromium-browser"
+      - PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
+  testWeb:
+    image: philpl/alpine-chrome-node:latest
+    command:
+      - yarn
+    args:
+      - test:web
+  testNative:
+    image: philpl/alpine-chrome-node:latest
```

Comment:
Can we use a public image? Just thinking if something happens it'll be really hard to fix without your context

Reply:
Nope; I'll eventually move the Docker image somewhere else and this literally just includes `USER root` but there's no Docker image that works perfectly out of the box for us :disappointed:

Reply:
Removed the integration tests for now which enables us to use the default Docker image for node. They don't seem to be running anyway.

Also removed native/primitives for now. They don't seem to work due to a react-native change and our rather weird Jest config 😅